basyx.aasrepository/basyx.aasrepository-http/src/test/java/org/eclipse/digitaltwin/basyx/aasrepository/http/AasRepositoryHTTPSuite.java

```diff
@@ -387,4 +458,46 @@ private String getPaginatedSingleSMReferenceJson() throws FileNotFoundException,
 		return BaSyxHttpTestUtils.readJSONStringFromClasspath("PaginatedSingleSMReference.json");
 	}
 
+	private CloseableHttpResponse uploadThumbnail(String aasId) throws IOException {
+		CloseableHttpClient client = HttpClients.createDefault();
+
+
+		java.io.File file = ResourceUtils.getFile("src/test/resources/" + THUMBNAIL_FILE_PATH);
+
+		HttpPut putRequest = createPutRequestWithFile(aasId, THUMBNAIL_FILE_PATH, file);
+
+		return executePutRequest(client, putRequest);
+	}
+
+	private CloseableHttpResponse executePutRequest(CloseableHttpClient client, HttpPut putRequest) throws IOException {
```

Comment:
Why not use https://github.com/zhangzai123/basyx-java-server-sdk/blob/main/basyx.common/basyx.http/src/test/java/org/eclipse/digitaltwin/basyx/http/serialization/BaSyxHttpTestUtils.java#L140

Reply:
this put request, a file must be uploaded. In the method of BaSyxHttpUtils#L140, you cannot upload file, only string.

Reply:
IN BaSyxHttpUtils verlegen